src/pytorch_lightning/strategies/single_hpu.py

```diff
@@ -79,10 +82,22 @@ def setup_optimizers(self, trainer: "pl.Trainer") -> None:
     def model_to_device(self) -> None:
         self.model.to(self.root_device)  # type: ignore
 
-    def training_step_end(self, step_output: STEP_OUTPUT) -> STEP_OUTPUT:
-        # Break lazy accumulation of graph after every step
+    def on_after_backward(self) -> None:
+        # Break lazy accumulation of graph after fwd+bwd
         htcore.mark_step()
-        return step_output
+
+    def optimizer_step(
+        self,
+        optimizer: Optimizer,
+        opt_idx: int,
+        closure: Callable[[], Any],
+        model: Optional[Union["pl.LightningModule", Module]] = None,
```

Comment:
Since you don't use any of them, this could just be

```suggestion
        *args: Any,
```

Reply:
I wanted to be compliant with the existing code

Reply:
What do you mean with compliant? How is the suggestion non-compliant?